### DISCUSSION TITLE
Hello, do you have an idea of ​​the pixel size of the images so that they do not take up too much space and at the same time they remain of good quality?

### DISCUSSION THREAD

**Discussion by apprenti1973** [opengisch/QField#3428] [Q&A]

**Comment by timbo-barnes** [comment]
Are you talking about photos taken with the device camera or rasters/basemaps within the project?

The short answer: is depends
For basemaps, it depends on the source data and needs of the user....obviously a topographical map will have a lower source resolution than high-resolution aerial imagery AND a user needing to get a general idea of their location from aerial photos will have a lower resolution need than a user needing to identify features from that aerial imagery.
